src/engine/runtime.js

```diff
@@ -769,16 +787,16 @@ class Runtime extends EventEmitter {
      * Emit run start/stop after each tick. Emits when `this.threads.length` goes
      * between non-zero and zero
      *
-     * @param {number} threadCount The new threadCount
+     * @param {number} nonMonitorThreadCount The new nonMonitorThreadCount
      */
-    _setThreadCount (threadCount) {
-        if (this._threadCount === 0 && threadCount > 0) {
+    _maybeEmitProjectRunStartOrStop (nonMonitorThreadCount) {
```

Comment:
Renamed. Nothing else was using _threadCount so I replaced it with _nonMonitorThreadCount which is specific to this function. Do you think we should still keep track of _threadCount?